src/wasm-stack.h

```diff
@@ -657,6 +670,41 @@ void BinaryenIRWriter<SubType>::visitHost(Host* curr) {
   emit(curr);
 }
 
+template<typename SubType> void BinaryenIRWriter<SubType>::visitTry(Try* curr) {
+  emit(curr);
+  // TODO: emit block contents directly, if possible
```

Comment:
i think this comment is not needed, as visitPossibleBlockContents does that? (it looks like we have some old comments like this before this PR)

Reply:
Yes, I copy-pasted this from `visitIf`... Deleted this in both `visitTry` and `visitIf`.